sites/fast-website/src/app/data/wave.data.ts

```diff
@@ -0,0 +1,102 @@
+// Tool to convert from SVG relative to absolute paths, super helpful
+// https://codepen.io/leaverou/pen/RmwzKv
```

Comment:
In case this goes away, can you add in a little more context like that the conversion logic is by Snap.svg?

Reply:
Sure thing - this comment was from Parker's original deliverable. I'll flesh out the comment to point to Snap.svg.